test/integration/test_simplex.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+import numpy as np
+
+import porepy as pp
+
+
+class TestFaceTags(unittest.TestCase):
+    def test_face_tags_from_gmsh(self):
```

Comment:
Is this different from test/unit/test_tag_gmsh? Please explain what is tested.

Reply:
see previous comment